plugins/triple_oscillator/TripleOscillator.cpp

```diff
@@ -306,6 +324,9 @@ void TripleOscillator::playNote( NotePlayHandle * _n,
 		for( int i = NUM_OF_OSCILLATORS - 1; i >= 0; --i )
 		{
 
+			float phaseRandL = fastRandf( 1 ) * m_osc[i]->m_phaseRand;
+			float phaseRandR = fastRandf( 1 ) * m_osc[i]->m_phaseRand;
```

Comment:
FYI, a periods corresponds to 2π in phase.

Reply:
Never mind, `Oscillator` uses unusual convention(a period = phase of 1). So this code is fine as-is.

Reply:
Just a random thought: would it be better to use -0.5 \~ 0.5 instead of 0 \~ 1?